jax/_src/lax/lax.py

```diff
@@ -527,20 +527,47 @@ def concatenate(operands: Sequence[Array], dimension: int) -> Array:
   """
   return concatenate_p.bind(*operands, dimension=dimension)
 
-Precision = xla_client.PrecisionConfig.Precision
-Precision.__str__ = lambda precision: precision.name  # type: ignore
+class Precision(xla_client.PrecisionConfig.Precision):  # type: ignore
+  """Precision enum for lax functions
+
+  The `precision` argument to JAX functions generally controls the tradeoff
+  between speed and accuracy for array computations on the TPU backend. It
+  currently does not affect computation on CPU or GPU. Members are:
+
+  DEFAULT:
+    Fastest mode, but least accurate. Performs computations in bfloat16.
+    Aliases: ``'default'``, ``'fastest'``, ``'bfloat16'``.
+  HIGH:
+    Slower but more accurate. Performs computations in 3 bfloat16 passes,
+    or using tensorfloat32 where available. Aliases: ``'high'`,
+    ``'bfloat16_3x'``, ``'tensorfloat32'``.
+  HIGHEST:
+    Slowest but most accurate. Performs computations in float32.
```

Comment:
I suspect this description is misleading if using 64-bit (really ef57) values on TPU. But I'm not sure.

Reply:
Perhaps "performs computations at the highest available precision (float32 or float64, as applicable)"?